vumi/scripts/vumi_list_messages.py

```diff
@@ -0,0 +1,97 @@
+#!/usr/bin/env python
+# -*- test-case-name: vumi.scripts.tests.test_vumi_list_messages -*-
+
+import sys
+
+from twisted.internet.defer import inlineCallbacks
+from twisted.internet.task import react
+from twisted.python import usage
+
+from vumi.components.message_store import MessageStore
+from vumi.persist.txriak_manager import TxRiakManager
+
+
+class Options(usage.Options):
+    optParameters = [
+        ["batch", None, None,
+         "Batch identifier to list messages for."],
+        ["bucket-prefix", "b", None,
+         "The bucket prefix for the Riak manager."],
+        ["direction", None, None,
+         "Message direction. Valid values are `inbound' and `outbound'."],
```

Comment:
There are a bunch of things that use starting-backtick-ending-apostrophe for quoting values like that, especially in usage and manpages. I can change it to apostrophes on both sides if you prefer.